From a6592ced9fba51bbe02f28b163ae5a8e030d09da Mon Sep 17 00:00:00 2001
From: tic_chen <tic_chen@gemteks.com>
Date: Tue, 20 Jan 2015 12:23:19 +0800
Subject: fix build problem under linux.

---
 common/common.mk  | 4 ++--
 weesetup/Makefile | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/common/common.mk b/common/common.mk
index 41d768f..41aea95 100644
--- a/common/common.mk
+++ b/common/common.mk
@@ -55,8 +55,8 @@ SYSSTR   = w32
 endif
 
 ifneq ($(findstring Linux,$(OSTYPE)),)
-CFLAGS   += -DLINUX
-AFLAGS   += -DLINUX
+CFLAGS   += -DLINUX -m32
+AFLAGS   += -DLINUX -m32
 endif
 
 USE_ZIP  ?= y
diff --git a/weesetup/Makefile b/weesetup/Makefile
index e5baf1c..059e3bb 100644
--- a/weesetup/Makefile
+++ b/weesetup/Makefile
@@ -41,6 +41,6 @@ weesetup$(EXEEXT): weesetup.o xdio.o utils.o
 	$(CC) $(CFLAGS) -s -obin/$@ $^
 	rm *.o *.d mbr.h
 mbr.h: wee63.mbr $(BIN2H_DEPS)
-	./$(BIN2H_EXEC) $< $@ wee63_mbr
+	$(BIN2H_EXEC) $< $@ wee63_mbr
 wee63.mbr: nt6mbr.bin ../wee/wee63.mbr
-	./mkmbr
\ No newline at end of file
+	sh mkmbr
-- 
2.2.2

